### PR TITLE
Add numerals to dictionaries and finish fixing speller

### PR DESCRIPTION
**Pull request by mr-martian**
Numerals (`/[0-9]+/`, analysis `<num>`) are relevant in nearly all
languages, so add those in all 3 formats.

The speller fixes in #115 didn't actually result in a working makefile
because the files it depended on weren't included.

### CODE REVIEW

**Comment by mr-martian** [comment]
I left the speller code behind `--enable-ospell` because I wasn't sure whether we wanted it compiling by default or not.

**Comment by mr-martian** [comment]
... and I forgot that in addition to adding the speller, `--enable-ospell` also checks if `hfst-ospell` is installed, even though it's not needed for compilation.

**Comment by mr-martian** [comment]
tests are failing because of a bugfix that hasn't made it into the nightly build yet.

**Comment by jonorthwash** [comment]
> I left the speller code behind `--enable-ospell` because I wasn't sure whether we wanted it compiling by default or not.

Is there any reason not to?

> ... and I forgot that in addition to adding the speller, `--enable-ospell` also checks if `hfst-ospell` is installed, even though it's not needed for compilation.

Why?  If someone wants to use the speller, there should just be documentation about how to do that, including what packages to install.  I see no problem with checking on runtime needs for compilation.

@TinoDidriksen, thoughts?

**Comment by mr-martian** [comment]
I was thinking that it makes the output of `make` nearly illegible because the output of `editdist` takes up so much space, but then I realized that I could just reformat the output of `editdist`.

The current setup of this PR is that there is no configure check and it just compiles the speller regardless.

**Comment by TinoDidriksen** [comment]
> > ... and I forgot that in addition to adding the speller, `--enable-ospell` also checks if `hfst-ospell` is installed, even though it's not needed for compilation.
> 
> Why? If someone wants to use the speller, there should just be documentation about how to do that, including what packages to install. I see no problem with checking on runtime needs for compilation.

If it's not actually required for compilation, it should not be required by configure.ac. We want builders to run in absolutely minimal configurations, so any superfluous package is bad.

What it could do is print out a "you will need hfst-ospell or another zhfst capable tool to use the built speller" message, but it should not block the build.

**Comment by mr-martian** [comment]
Ok, now it will warn but not error if `hfst-ospell` is not installed.

**Review by TinoDidriksen — APPROVED** [review]

**Review by jonorthwash — APPROVED** [review]

File: hfst-language-module/speller/index.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <!-- TODO: locale may need to be changed to 2-letter ISO 639-1 code -->
+    <title>Apertium morphological analyser-based speller ({{languageName}})</title>
+    <description>
+      A spellchecker for {{languageName}}.
+    </description>
+    <version vcsrev="GT_REVISION">GT_VERSION</version>
+    <date>DATE</date>
+    <producer>Apertium contributors</producer>
+    <contact email="feedback@divvun.no"
+        website="http://divvun.no"/>
```
Comment (TinoDidriksen) [inline hfst-language-module/speller/index.xml:14]: Upon review, maybe those 2 should be changed. And GT_VERSION isn't a thing for us - should be `@APERTIUM_AUTO_VERSION@`

**Review by TinoDidriksen — APPROVED** [review]

**Pull request merged**